project/tasks/collaborate.rst

```diff
@@ -0,0 +1,93 @@
+====================
+Collaborate on Tasks
+====================
+
+Being able to collaborate virtually, but with the same effect as if people were in the same room,
+increases your productivity. It results in faster execution of processes,
+brainstorming and decision made easier while also allowing the support of outside contributions.
+
+Add followers to a project
+==========================
+
+| *Edit* the project to add followers.
+| Followers are notified of any changes: tasks sliding from one stage to another, conversations
+  taking place, etc.
```

Comment:
```suggestion
| Followers are notified for any change: tasks sliding from one stage to another, conversations
  taking place, etc.
```

Reply:
From my research 'notified of' is correct, and because I will give more than example, I rather keep 'changes'.